dbms/src/Storages/DeltaMerge/Remote/RemoteReadTask.cpp

```diff
@@ -421,36 +400,32 @@ RemoteSegmentReadTaskPtr RemoteSegmentReadTask::buildFrom(
         table_id,
         proto);
 
+    // Note: The segment_snap here is not readable.
```

Comment:
What do you mean by "not readable"? Can you explain more specifically?

Reply:
We should not read from the segment_snap at this moment, because it has not been connected to an underlying storage.

I will enhance the comment when cherry picking to master.